hydra-cluster/src/HydraNode.hs

```diff
@@ -275,16 +280,20 @@ withHydraNode tracer chainConfig workDir hydraNodeId hydraSKey hydraVKeys allNod
               }
       let p =
             ( hydraNodeProcess $
-                defaultOptions
-                  { nodeId = fromIntegral hydraNodeId
+                Options
+                  { verbosity = Verbose "HydraNode"
+                  , nodeId = fromIntegral hydraNodeId
+                  , host = "127.0.0.1"
                   , port = fromIntegral $ 5000 + hydraNodeId
+                  , peers
+                  , apiHost = "127.0.0.1"
```

Comment:
`defaultOptions` was removed from the source code, and thus, the unspecified default options have been inlined.